<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab2SocketClient</class>
 <widget class="QWidget" name="Tab2SocketClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,1">
     <item>
      <widget class="QLabel" name="pLabel">
       <property name="text">
        <string>수신 데이터</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBRecvDataClr">
       <property name="text">
        <string>수신 삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBServerConnect">
       <property name="text">
        <string>서버 연결</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="pTERecvData"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,6,2">
     <item>
      <widget class="QLineEdit" name="pLERecvId"/>
     </item>
     <item>
      <widget class="QLineEdit" name="pLESendData"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBSendButton">
       <property name="text">
        <string>송신</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
